manifests/init.pp

```diff
@@ -504,7 +504,7 @@
       hasrestart => $trap_service_hasrestart,
       require    => [ Package['snmpd'], File['var-net-snmp'], ],
     }
-  } elsif $::osfamily == 'Suse' {
+  } elsif $::operatingsystem =~ /(?i:opensuse|suse)/ {
```

Comment:
I use $::operatingsystem because OpenSUSE Tumbleweed reports $::osfamily as 'Linux'.  OpenSUSE Leap *does* behave correctly however and returns $::osfamily as 'Suse' which makes supporting different SUSE releases a bit difficult.